hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/builder/JpaTargetCreate.java

```diff
@@ -38,6 +47,10 @@ public JpaTarget build() {
             target.setName(name);
         }
 
+        if (Objects.nonNull(targetTypeId)){
```

Comment:
yes by default target type value is null but we want to avoid assigning null because by default it will set null just like any optional field will be set to null if the value is not provided

Reply:
Normal `!= null` should be enough no need for Objects method call